editor.planx.uk/src/@planx/components/ExternalPortal/Editor.tsx

```diff
@@ -51,20 +155,78 @@ const ExternalPortalForm: React.FC<{
             flow that it references.
           </span>
         </ModalSectionContent>
-        <ModalSectionContent title="Pick a flow">
-          <select
+        <ModalSectionContent
+          key={"team-section"}
+          title="Select a team (optional)"
+        >
+          <SelectMultiple
+            id="team-select"
+            onChange={(_options, event) => {
+              filterFlows(event);
+            }}
+            value={teamArray}
+            options={uniqueTeamArray}
+            renderOption={(props, option, { selected }) => {
+              return (
+                <ListItem {...props} key={`${option}-listitem`}>
+                  <CustomCheckbox
+                    key={`${option}-checkbox`}
+                    aria-hidden="true"
+                    className={selected ? "selected" : ""}
+                  />
+                  {option}
+                </ListItem>
+              );
+            }}
+            placeholder=""
+          />
+        </ModalSectionContent>
+        <ModalSectionContent key={"flow-section"} title="Pick a flow">
+          <Autocomplete
             data-testid="flowId"
-            name="flowId"
-            value={formik.values.flowId}
-            onChange={formik.handleChange}
-          >
-            {!id && <option value="" />}
-            {flows.map((flow) => (
-              <option key={flow.id} value={flow.id}>
-                {flow.text}
-              </option>
-            ))}
-          </select>
+            id="flowId"
+            role="status"
+            aria-atomic={true}
+            aria-live="polite"
+            fullWidth
+            popupIcon={PopupIcon}
+            ListboxProps={{
+              sx: (theme) => ({
+                paddingY: 0,
+                backgroundColor: theme.palette.background.default,
+              }),
+            }}
+            value={
+              flows.find((flow: Flow) => flow.id === formik.values.flowId) ||
+              flows[0]
+            }
+            onChange={(_event, newValue: Flow) => {
+              formik.setFieldValue("flowId", newValue.id);
+            }}
+            options={
+              flows &&
+              flows.filter((flow) => {
```

Comment:
You can write this more succinctly like this: 

```suggestion
              flows?.filter((flow) => {
```